test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -11212,5 +11212,25 @@ TEST_F(AtenXlaTensorTest, TestRoll) {
   ExpectCounterChanged("xla::roll", cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestViewIsAliasOf) {
+  auto a = torch::empty(4, torch::TensorOptions(torch::kFloat));
```

Comment:
can we spell out `torch::tensor` in the test? I found `auto` hard to read sometime 😄

Reply:
Sure. I'm a lazy person so I'm a big fans of `auto`.